reviewing/dependencies-hierarchy/src/buildDependenciesHierarchy.ts

```diff
@@ -47,13 +48,13 @@ export async function buildDependenciesHierarchy (
     throw new TypeError('opts.lockfileDir is required')
   }
   const modulesDir = await realpathMissing(path.join(maybeOpts.lockfileDir, maybeOpts.modulesDir ?? 'node_modules'))
-  const modules = await readModulesManifest(modulesDir)
+  const modules = maybeOpts.lockfileOnly ? null : await readModulesManifest(modulesDir)
```

Comment:
I am not sure this change is needed.

Reply:
I thought it might help performance, but it's not an issue.